### PR TITLE
Added examples to Path.GetRelativePath docs

### PR DESCRIPTION
**Pull request by peteraritchie**
## Summary

Describe your changes here.

Fixes #Issue_Number (if available)

### CODE REVIEW

File: xml/System.IO/Path.xml [hunk 1]
```diff
@@ -2023,6 +2023,22 @@ Unlike <xref:System.IO.Path.GetTempFileName%2A>, <xref:System.IO.Path.GetRandomF
 
 Paths are resolved by calling the <xref:System.IO.Path.GetFullPath%2A> method before calculating the difference. The method uses the default file path comparison for the current platform (<xref:System.StringComparison.OrdinalIgnoreCase?displayProperty=nameWithType> for Windows and MacOs, <xref:System.StringComparison.Ordinal?displayProperty=nameWithType> for Linux.
 
+## Examples
+ The following code shows how to call the <xref:System.IO.Path.GetRelativePath%2A> method.
+
+```vb
+		' "C:/Program Files/Microsoft" relative to "C:/Program Files/Common Files" is "../Microsoft"
+		Debug.Assert(("../Microsoft" = Path.GetRelativePath("C:/Program Files/Common Files", "C:/Program Files/Microsoft")))
```
Comment (Jozkee) [inline xml/System.IO/Path.xml:2031]: I usually don't see examples displaying behavior with assertions; write to console instead.
```suggestion
		Console.WriteLine(Path.GetRelativePath("C:/Program Files/Common Files", "C:/Program Files/Microsoft"))
```

File: xml/System.IO/Path.xml [hunk 2]
```diff
@@ -2023,6 +2023,22 @@ Unlike <xref:System.IO.Path.GetTempFileName%2A>, <xref:System.IO.Path.GetRandomF
...
+		' "C:/Program Files/Microsoft" relative to "C:/Program Files/" is "Microsoft"
+		Debug.Assert(("Microsoft" = Path.GetRelativePath("C:/Program Files/", "C:/Program Files/Microsoft")))
+```
+
+```csharp
```
Comment (Jozkee) [inline xml/System.IO/Path.xml:2036]: Why is VB example indented but not this one?

File: xml/System.IO/Path.xml [hunk 3]
```diff
@@ -2023,6 +2023,22 @@ Unlike <xref:System.IO.Path.GetTempFileName%2A>, <xref:System.IO.Path.GetRandomF
...
+// "C:/Program Files/Microsoft" relative to "C:/Program Files/Common Files" is "../Microsoft"
+Debug.Assert("../Microsoft" == Path.GetRelativePath("C:/Program Files/Common Files", "C:/Program Files/Microsoft"));
+// "C:/Program Files/Microsoft" relative to "C:/Program Files/" is "Microsoft"
+Debug.Assert("Microsoft" == Path.GetRelativePath("C:/Program Files/", "C:/Program Files/Microsoft"));
+```
```
Comment (Jozkee) [inline xml/System.IO/Path.xml:2041]: nit: consider adding them to the snippets folder e.g:
https://github.com/dotnet/dotnet-api-docs/blob/061c06aaa286ab20ea139648d498efd86ea98014/xml/System.IO/Path.xml#L109-L111

File: xml/System.IO/Path.xml (hunk 2 above)
Comment (peteraritchie) [reply xml/System.IO/Path.xml]: Good catch.

File: xml/System.IO/Path.xml (hunk 3 above)
Comment (peteraritchie) [reply xml/System.IO/Path.xml]: Done

File: snippets/visualbasic/System.IO/Path/GetRelativePath/GetRelativePath.vb [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+        ' This code produces output similar to the following:
+        '
+        ' ../Microsoft
+        ' Microsoft
+        ' </Snippet1>
+
+    End Sub
+
+End Module
+'</snippet1>
```
Comment (Jozkee) [inline snippets/visualbasic/System.IO/Path/GetRelativePath/GetRelativePath.vb:22]:
```suggestion
```

**Review by Jozkee — APPROVED** [review]
Otherwise, LGTM; thanks!

**Pull request merged**